apps/stellar/stellar.cpp

```diff
@@ -581,6 +581,9 @@ void _setParser(ArgumentParser & parser)
 
 // TODO(holtgrew): Move this into a SeqAn misc module.
 
+// not supported anymore in vc2015
+// https://msdn.microsoft.com/en-us/library/bb531344.aspx
+#if _MSC_VER < 1900
```

Comment:
Hmm _set_output_format() was necessary in older VS and is now deprecated in VS2015, right? If so, ```#ifdef PLATFORM_WINDOWS_VS``` becomes ```#if _MSC_VER < 1900```.